Hunting Queries/SecurityEvent/RareProcessPath.yaml

```diff
@@ -0,0 +1,66 @@
+id: 0ff22697-dc58-4623-b844-a767629840cd
+name: Rare Process Path
+description: |
+  'Identifies when a process is running from a rare path. This could indicate malicious or unexpected activity as attacks 
+  often try to use common process names running from non-standard locations'
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Execution
+query: |
+
+  let end = startofday(now());
+  let start = end - 7d;
+  let processEvents=
+  SecurityEvent
+  | where TimeGenerated >= start and TimeGenerated <= end
+  | where EventID==4688
+  // excluding well known processes
```

Comment:
Have you also considered excluding processes that only run once in the time window? They will always have a rare path by this measurement but they might be running from a completely expected path.

Reply:
Yeah, but we don't want to exclude those either but I meant to make the query 8 days so we cover items that are run weekly as part of a scheduled task or deployment that only runs once. I will change this to 8 days. If it is a task that runs less often, the user will have to tune this.